tests/queries/0_stateless/01737_clickhouse_server_wait_server_pool_long.sh

```diff
@@ -26,7 +29,8 @@ function cleanup()
     echo "Test failed. Server log:"
     cat clickhouse-server.log
     rm -f clickhouse-server.log
-
+    rm -fr $CUR_DIR/01737_clickhouse
```

Comment:
This looks extremely dangerous.
What if `$CUR_DIR` contains whitespaces?

Reply:
i'll fix